From 23a47f2947236d29d836aa73bf93185917bc9b26 Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Tue, 5 Jun 2018 13:31:55 -0400
Subject: [PATCH] TRITON-461 triton services still on nodejs 0.10.x should at
 least use the latest x Reviewed by: Trent Mick <trentm@gmail.com> Approved
 by: Trent Mick <trentm@gmail.com>

---
 Makefile | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index b88a3bf..21f4d95 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright (c) 2016, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 #
@@ -21,8 +21,7 @@ include ./tools/mk/Makefile.defs
 # binary as part of the build process.  Other options are possible -- it depends
 # on the need of your repository.
 #
-# NODE_PREBUILT_VERSION=v4.4.0
-NODE_PREBUILT_VERSION=v0.10.43
+NODE_PREBUILT_VERSION=v0.10.48
 ifeq ($(shell uname -s),SunOS)
 	# Allow building on a SmartOS image other than sdc-*-multiarch 15.4.1.
 	NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
-- 
2.21.0

